packages/server/lib/util/electron_app.js

```diff
@@ -10,6 +10,43 @@ const scale = () => {
   }
 }
 
+/**
+ * Parses a single Electron launch argument. If argument has value "true" it is converted
+ * into a boolean.
+ *
+ * @param {string} token Token to parse, like "disable-renderer-backgrounding=true"
+ */
+const parseLaunchArgument = (token) => {
+  if (token.includes('=')) {
+    let [key, value] = token.split('=')
+
+    if (value === 'true') {
+      value = true
+    }
+
+    return [key, value]
+  }
+
+  return [token, undefined]
+}
+/**
+ * Users can specify additional Electron launch flags via an environment variable.
+ * This function parses that variable's string into an object.
+ * @param {string} s Input string with values likes "foo bar=baz disable-renderer-backgrounding=true"
+*/
+const parseElectronLaunchArguments = (s) => {
+  const tokens = s.split(' ')
```

Comment:
this will break if the argument contains a space, but idk how likely it is. how likely do you think it is that a user will need to pass arguments containing spaces to Electron?

Reply:
I don't think any Electron or Chrome launch args can have  spaceshttps://electronjs.org/docs/api/chrome-command-line-switches/history